test/performance/odp_bench_packet.c

```diff
@@ -587,6 +713,18 @@ static int bench_packet_data(void)
 	return i;
 }
 
+static int bench_packet_data_seg_len(void)
+{
+	odp_packet_t *pkt_tbl = gbl_args->pkt_tbl;
+	uint32_t *output_tbl = gbl_args->output_tbl;
+	int i;
+
+	for (i = 0; i < TEST_REPEAT_COUNT; i++)
+		gbl_args->ptr_tbl[i] = odp_packet_data_seg_len(pkt_tbl[i],
```

Comment:
Better to increment a length variable to avoid extra data movement, like packet len:
ret += odp_packet_len()

Reply:
odp_packet_data_seg_len() returns a data pointer, not the length.